tests/Integration/AbstractTechproductsTest.php

```diff
@@ -55,6 +55,22 @@ public static function setUpBeforeClass(): void
         $ping = self::$client->createPing();
         self::$client->ping($ping);
 
+        // disable automatic commits for update tests
+        $request = new Request();
```

Comment:
```php
$query = self::$client->createApi([
    'version' => Request::API_V2,
    'handler' => 'config',
    'rawdata' => json_encode([
            'set-property' => [
                'updateHandler.autoCommit.maxDocs' => -1,
                'updateHandler.autoCommit.maxTime' => -1,
                'updateHandler.autoSoftCommit.maxDocs' => -1,
                'updateHandler.autoSoftCommit.maxTime' => -1,
            ],
        ]),
]);
$response = self::$client->execute($query);
```

Reply:
I had to use `API_V1`. V2 only has a config endpoint for collections, not for cores.

I've set `openSearcher` explicitly as well and included a check that reads the properties back and ensures they are what we want them to be.